sig-aws/README.md

```diff
@@ -37,5 +37,8 @@ Note that the links to display team membership will only work if you are a membe
 | @kubernetes/sig-aws-misc | [link](https://github.com/orgs/kubernetes/teams/sig-aws-misc) | General Discussion |
 
 <!-- BEGIN CUSTOM CONTENT -->
+## Contributing
+Feel free to join our Meetings and help us growing. We depend on new people becoming members and regular code contributors.
```

Comment:
Help us grow? 

We should have a contributing in this directory as well.

Reply:
I tried to mix contribution invitation + meeting invitation.

Will split it into two Parts. A CONTRIBUTING.md and some Meeting Invitation.